src/Drivers/check_spo.cpp

```diff
@@ -73,9 +69,6 @@ int main(int argc, char **argv)
     case 'i': // number of MC steps
       nsteps = atoi(optarg);
       break;
-    case 's': // random seed
```

Comment:
This should probably stay, and get hooked up to the random generator.  It looks like the code generates random electron positions.   Any code that uses random numbers should have a way to specify the seed.